packages/account/src/BiconomySmartAccount.ts

```diff
@@ -256,6 +256,37 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
     return "0x";
   }
 
+  private async getNonce(): Promise<BigNumber> {
+    let nonce = BigNumber.from(0);
+    try {
+      nonce = await this.nonce();
+    } catch (error) {
+      // Not throwing this error as nonce would be 0 if this.nonce() throw exception, which is expected flow for undeployed account
```

Comment:
Add a logger here.
"Error while getting nonce for the account. This is expected for undeployed accounts"